src/main/webapp/file.xhtml

```diff
@@ -552,11 +553,16 @@
                                                         <td>#{FilePage.file.publicationDateFormattedYYYYMMDD}
                                                         </td>
                                                     </tr>
-                                                    <tr id="fileEPublicationDateBlock" jsf:rendered="#{FilePage.file.released or !(empty FilePage.file.embargo)}">
+                                                    <tr id="fileEPublicationDateBlock" jsf:rendered="#{FilePage.file.released or !(empty FilePage.file.embargo) or !(empty FilePage.file.retention)}">
                                                         <th scope="row">
                                                             #{bundle['file.metadataTab.fileMetadata.publicationDate.label']}
                                                         </th>
-                                                        <td>#{!(empty FilePage.file.embargo) ? FilePage.embargoPhrase: ''} #{!(empty FilePage.file.embargo) ? FilePage.file.embargo.dateAvailable : FilePage.file.publicationDateFormattedYYYYMMDD}
+                                                        <td>#{!(empty FilePage.file.embargo) ? FilePage.embargoPhrase: ''} #{!(empty FilePage.file.embargo) ? FilePage.file.embargo.dateAvailable : ''}
+                                                            #{!(empty FilePage.file.embargo) and !(empty FilePage.file.retention) ? '; ': ''}
+                                                            #{!(empty FilePage.file.retention) ? FilePage.retentionPhrase: ''} #{!(empty FilePage.file.retention) ? FilePage.file.retention.dateUnavailable: ''}
```

Comment:
Can the next 3-4 lines be combined? They all check !(empty FilePage.file.retention) and add a sequence of 
`FilePage.retentionPhrase FilePage.file.retention.dateUnavailable , bundle['retention.enddateinfo']` if true and '' if not.

Reply:
I used `concat` to avoid that duplication